packages/lib/test/contracts/upgradeability/AdminUpgradeabilityProxy.test.js

```diff
@@ -130,8 +130,12 @@ contract('AdminUpgradeabilityProxy', ([_, admin, anotherAccount]) => {
           })
 
           it('uses the storage of the proxy', async function () {
-            // fetch the x value of Migratable at position 0 of the storage
-            const storedValue = await Proxy.at(this.proxyAddress).getStorageAt(1);
+            // storage layout should look as follows:
+            //  - 0: Initializable storage
+            //  - 1-50: Initailizable reserved storage (50 slots)
+            //  - 51: initializerRan
+            //  - 52: x
```

Comment:
Shouldn't it be...?

```
//  - 0: Initializable#initialized
//  - 1: Initializable#initializing
//  - 2-51: Initailizable#gap
//  - 52: x
```

Reply:
No, both `initialized` and `initializing` are `bool`, so they are packed together into the same storage slot. The test uses `InitiazableMock`, which has `bool initializerRan` and `uint256 x`, those are not packed due to `x` using the full slot.